ppr-ui/src/views/amendment/AmendRegistration.vue

```diff
@@ -593,12 +516,6 @@ export default defineComponent({
       return hasChanged
     }
 
-    const goToDashboard = (): void => {
-      // unset registration number
-      setRegistrationNumber(null)
```

Comment:
I updated the reset function called in dashboard to set the registration number to false.